RFPs.md

```diff
@@ -0,0 +1,19 @@
+Any submissions to the Sorobanathon are welcome but for those having trouble getting the cranium cauldion bubbling here are some guaranteed winners.
+
+- [ ] A written and/or video tutorial walking through the [soroban-example-dapp](https://github.com/stellar/soroban-example-dapp). [Related issue](https://github.com/stellar/soroban-example-dapp/issues/3)
+- [ ] Rock, paper, scissors game
+- [ ] Timed action & bid contract
+- [ ] Tic tac toe dapp
+- [ ] Something like https://sablier.finance/ or https://llamapay.io/ that allows you to "stream" payments
+- [ ] DAO contract examples. Anything from very simple to more complex
+- [ ] A lottery where people can win some NFT by entering
+- [ ] A wager pool maybe connected to a pseudo oracle like [Fortnite tracker](https://fortnitetracker.com/)
+- [ ] A calculator where the number literals are stored as NFTs, so a calculation would call an operand method, consume two NFTs representing numbers, and produce an NFT containing the result 🤪
+- [ ] One or more escrow contract examples
+- [ ] Zero knowledge proof exploratory concepts
+- [ ] A more complex and fully decentralized [hot potato](https://stellarhotpotato.tk/) game
+- [ ] A contract that you can send a payment to it and redistribute the payment randomly to the holders of a token or NFT
```

Comment:
This sentence is a bit funky.  Maybe: "A contract you can send a payment to that redistributes the payment randomly to the holders of a token or NFT"